test/format-aggregates.ts

```diff
@@ -0,0 +1,38 @@
+import { formatAggregates } from "../src/index";
```

Comment:
👏 

This test is not being run in CI. Need to rename to `format-aggregates.test.ts`

Reply:
Ah right! 🤦‍♂️ No wonder all tests passed when I ran it, I'll make changes.